recipes/nnetsauce/meta.yml

```diff
@@ -0,0 +1,58 @@
+{% set name = "nnetsauce" %}
+{% set version = "0.8.3" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/Techtonique/{{ name }}/tree/master/dist/{{ name }}-{{ version }}.tar.gz
+    sha256: 2163e76d6edc6339207b8ee89275ab0fc9c8846f1f6a6997c9fed5628ad5ae0e
+
+build:  
+    number: 0  
+    script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+    build:    
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+    host:
+        - python
+        - pip
+        - setuptools 
+        - cython
+    run:
```

Comment:
It looks like the dependencies under run need to have their versions constrained based on the `setup.py`:

https://github.com/Techtonique/nnetsauce/blob/master/setup.py#L238-L244

Reply:
Hmm... I don't get this one. Does it mean that I must specify the min. version in the recipe?

Reply:
You want whatever version restrictions are specified in the `setup.py` to match here. We want it so that the conda build is following whatever a user would get if they ran `python setup.py install` from the source directory or via pip. Also, I think the linter is going to complain if there is a space between the operator and version, so for example you would want `numpy >=1.14.0` rather than `numpy >= 1.14.0`. Currently the linter isn't checking the recipe because it is looking for `meta.yaml` and can't find it since the file is named `meta.yml`.